bondpy/package.xml

```diff
@@ -15,17 +15,20 @@
   <url type="repository">https://github.com/ros/bond_core</url>
   <author>Stuart Glaser</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <test_depend>ament_copyright</test_depend>
+  <test_depend>ament_flake8</test_depend>
+  <test_depend>ament_pep257</test_depend>
+  <test_depend>python3-pytest</test_depend>
 
   <build_depend>bond</build_depend>
   <build_depend>rospy</build_depend>
   <build_depend>smclib</build_depend>
 
-  <exec_depend>rospy</exec_depend>
+  <exec_depend>rclpy</exec_depend>
   <exec_depend>smclib</exec_depend>
   <exec_depend>uuid</exec_depend>
 
   <export>
-    <architecture_independent/>
+    <build_type>ament_python</build_type>
   </export>
-</package>
+</package>
```

Comment:
Please add a trailing newline.

Reply:
Changed in [57fd471](https://github.com/ros/bond_core/pull/89/commits/57fd47196927dd08479548c6495c1883cf621561)